runtime/vm/jvminit.c

```diff
@@ -3926,6 +3926,9 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 	vm->checkpointState.lastRestoreTimeMillis = -1;
 	/* Its unclear if we need an option for this, so we can keep the init here for the time being */
 	vm->checkpointState.maxRetryForNotCheckpointSafe = 100;
+	vm->checkpointState.sleepNanosecondsForNotCheckpointSafe = 10000;
```

Comment:
I think you only need `sleepNanosecondsForNotCheckpointSafe`. But this can be milliseconds instead.